packages/workspace/src/utilities/buildable-libs-utils.ts

```diff
@@ -16,12 +16,38 @@ import { ensureTypescript } from './typescript';
 
 let tsModule: typeof import('typescript');
 
-function isBuildable(target: string, node: ProjectGraphProjectNode): boolean {
-  return (
-    node.data.targets &&
-    node.data.targets[target] &&
-    node.data.targets[target].executor !== ''
-  );
+function isBuildable(node: ProjectGraphProjectNode): string {
+  let buildTargetName: string | undefined = undefined;
+  const listOfBuildExecutors = [
```

Comment:
We should add a `TODO(v17)` so we don't forget to clean this up when we remove the `@nrwl/*` scope.

Reply:
Done!